pages/docs/tutorials/native/mapping-primitive-data-types-from-c.md

```diff
@@ -44,6 +44,7 @@ We create a `lib.h` file to see how C functions are mapped into Kotlin:
 #define LIB2_H_INCLUDED
 
 void ints(char c, short d, int e, long f);
+void uints(unsigned char c, unsigned short d, unsigned int e, unsigned long f);
```

Comment:
is 'uints' right?

Reply:
that is correct, it is the name of a function